index.bs

```diff
@@ -395,6 +395,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: User consent means the user agrees with what they are being asked, i.e., it encompasses reading and understanding prompts.
     An [=authorization gesture=] is a [=ceremony=] component often employed to indicate [=user consent=].
 
+: <dfn>User Handle</dfn>
+:: The user handle is specified by a [=[RP]=] and is a unique identifier for a user account with that [=[RP]=]. A user handle is
+    an opaque byte array with a maximum size of 64 bytes.
+:: The user handle is not meant to be displayed to the user, but is used by the [=[RP]=] to control the number of credentials - an
```

Comment:
I think you're using this `::` to get a paragraph break, in which case use the following instead:

```
:: The user handle is specified by a [=[RP]=] and is a unique identifier for a user account with that [=[RP]=]. A user handle is
    an opaque byte array with a maximum size of 64 bytes.

    The user handle...
```

Reply:
Good catch! Fixed.